modules/client/src/controllers/ResolveConditionController.ts

```diff
@@ -207,8 +212,9 @@ export class ResolveConditionController extends AbstractController {
 
     return {
       appId,
-      freeBalance: await this.connext.getFreeBalance(assetId),
+      sender,
       paymentId,
+      meta,
```

Comment:
i feel like we should keep in the free balance to make this change purely additive instead of breaking

Reply:
I really don't like the pattern of returning free balance like this so I'm trying to remove it. I think now is the best time to break things like this.